SimMuon/Configuration/python/customizeMuonDigi.py

```diff
@@ -0,0 +1,353 @@
+        input = cms.VInputTag(),
+        type = cms.string('PCaloHit'),
+        subdets = cms.vstring()
+    ),
+    mixHepMC = cms.PSet(
+        input = cms.VInputTag(cms.InputTag("generatorSmeared"),cms.InputTag("generator")),
+        makeCrossingFrame = cms.untracked.bool(True),
+        type = cms.string('HepMCProduct')
+    ),
+    mixVertices = cms.PSet(
+        input = cms.VInputTag(cms.InputTag("g4SimHits")),
+        makeCrossingFrame = cms.untracked.bool(True),
+        type = cms.string('SimVertex')
+    ),
+    mixSH = cms.PSet(
+        crossingFrames = cms.untracked.vstring(
+            'MuonCSCHits',
+            'MuonDTHits',
+            'MuonRPCHits'
+            ),
+        input = cms.VInputTag(
+            cms.InputTag("g4SimHits","MuonCSCHits"),
+            cms.InputTag("g4SimHits","MuonDTHits"),
+            cms.InputTag("g4SimHits","MuonRPCHits")),
+        type = cms.string('PSimHit'),
+        subdets = cms.vstring(
+            'MuonCSCHits',
+            'MuonDTHits',
+            'MuonRPCHits'
+            )
+    ),
+    mixTracks = cms.PSet(
+        input = cms.VInputTag(cms.InputTag("g4SimHits")),
+        makeCrossingFrame = cms.untracked.bool(True),
+        type = cms.string('SimTrack')
+    )
+)
+
+
+# Customize process.mix to be used for running muon (DT, CSC, RPC) digi only.
+#  - remove non-muon digitizers that are now run as part of mixing process
+#  - delete all the digitizers' aliases.
+#  - drop unnecessary mixObjects
+def customize_mix_muon_only(process):
+    process.mix.digitizers = digitizers = cms.PSet()
+    digi_aliases = filter(lambda n: 'Digi' in n, process.aliases.keys())
+    for a in digi_aliases: process.__delattr__(a)
+    process.mix.mixObjects = mixObjects_dt_csc_rpc
+    return process
+
+
+# customize process.mix by appending GEM SimHit collection definitions to mix.mixObjects.mixSH
+def customize_mix_addGEM(process):
+    process.mix.mixObjects.mixSH.crossingFrames.append('MuonGEMHits')
+    process.mix.mixObjects.mixSH.input.append(cms.InputTag("g4SimHits","MuonGEMHits"))
+    process.mix.mixObjects.mixSH.subdets.append('MuonGEMHits')
+    return process
+
+
+# customize process.mix by appending ME0 SimHit collection definitions to mix.mixObjects.mixSH
+def customize_mix_addME0(process):
+    process.mix.mixObjects.mixSH.crossingFrames.append('MuonME0Hits')
+    process.mix.mixObjects.mixSH.input.append(cms.InputTag("g4SimHits","MuonME0Hits"))
+    process.mix.mixObjects.mixSH.subdets.append('MuonME0Hits')
+    return process
+
+
+## load the digitizer and pad producer
+def load_GEM_digitizers(process):
+    process.load('SimMuon.GEMDigitizer.muonGEMDigi_cff')
+    return process
+
+
+## load the digitizer and pad producer
+def load_ME0_digitizers(process):
+    process.load('SimMuon.GEMDigitizer.muonME0Digi_cff')
+    return process
+
+
+# Add simMuonGEMDigis to the list of modules served by RandomNumberGeneratorService
+def customize_random_GEMDigi(process):
```

Comment:
@kpedro88 If I use the construct 
<PRE>
# Add simMuonGEMDigis to the list of modules served by RandomNumberGeneratorService                                                                                                
def customize_random_GEMDigi(process):
    from IOMC.RandomEngine.IOMC_cff import RandomNumberGeneratorService as rndsv
    process.RandomNumberGeneratorService.simMuonGEMDigis = rndsv.simMuonGEMDigis
    return process
</PRE>

 in a driver command 

<PRE>
cmsDriver.py SingleMuPt100_pythia8_cfi \
--conditions auto:phase2_realistic -n 10 \
--era Phase2C2_timing \
--eventcontent FEVTDEBUG  \
-s GEN,SIM,DIGI:pdigi_valid \
--datatier GEN-SIM-DIGI \
--beamspot HLLHC \
--geometry Extended2023D4 \
--customise SimMuon/Configuration/customizeMuonDigi.customize_digi_addGEM_addME0_muon_only \
--python SingleMuPt100_2023tilted_GenSimFull.py \
--no_exec --fileout file:step1.root
</PRE>

then the driver crashes with 

<PRE>
...
    process = customize_random_GEMDigi(process)
  File "/afs/cern.ch/work/d/dildick/public/GEM/ME0DigiIntegration/CMSSW_9_2_X_2017-05-08-2300/python/SimMuon/Configuration/customizeMuonDigi.py", line 65, in customize_random_GEMDigi
    process.RandomNumberGeneratorService.simMuonGEMDigis = rndsv.simMuonGEMDigis
AttributeError: 'Service' object has no attribute 'simMuonGEMDigis'
</PRE>

How do I load the Phase2 `RandomNumberGeneratorService` before calling the customization?